src/main/java/de/nplay/moderationbot/config/ConfigCommands.java

```diff
@@ -0,0 +1,59 @@
+package de.nplay.moderationbot.config;
+
+import com.github.kaktushose.jda.commands.annotations.Inject;
+import com.github.kaktushose.jda.commands.annotations.interactions.Choices;
+import com.github.kaktushose.jda.commands.annotations.interactions.Interaction;
+import com.github.kaktushose.jda.commands.annotations.interactions.Permissions;
+import com.github.kaktushose.jda.commands.annotations.interactions.SlashCommand;
+import com.github.kaktushose.jda.commands.dispatching.events.interactions.CommandEvent;
+import com.github.kaktushose.jda.commands.embeds.EmbedCache;
+import de.nplay.moderationbot.config.bot.BotConfig;
+import de.nplay.moderationbot.embeds.EmbedColors;
+import de.nplay.moderationbot.permissions.BotPermissions;
+import net.dv8tion.jda.api.Permission;
+
+@Interaction
+@Permissions(BotPermissions.ADMINISTRATOR)
+public class ConfigCommands {
+
+    @Inject
+    private EmbedCache embedCache;
+
+    @SlashCommand(value = "config set", desc = "Setzt eine Konfigurations-Variable", isGuildOnly = true, enabledFor = Permission.ADMINISTRATOR)
+    public void setConfig(CommandEvent event, @Choices({"spielersucheAusschlussRolle", "serverlogKanal"}) String name, String value) {
+        ConfigService.set(name, value);
+        event.reply(embedCache.getEmbed("configSet").injectValue("key", name).injectValue("value", value).injectValue("color", EmbedColors.SUCCESS));
+    }
```

Comment:
Choices should be human readable and mapped internally to the enum values (perhaps per type adapter?)

Alternatively one command per config value which is the easiest approach. As said before we wont get many more config values in the future, so this feature doesn't need to be super flexible